06.bowling_object/frame.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class Frame
+  attr_reader :shots, :score
+
+  def initialize(shots, frame_number)
+    @shots = shots
+    @frame_number = frame_number
+  end
+
+  def calculate_score(frames)
+    next_frame = next_frame(frames)
+    after_next_frame = after_next_frame(frames)
+
+    current_frame_score_sum = @shots.sum(&:point)
+    next_frame_score = next_frame.shots.map(&:point) if next_frame
```

Comment:
```suggestion
    next_frame_points = next_frame.shots.map(&:point) if next_frame
```

pointをmapして配列にしてるので、命名としてはこうでしょうね